bears/general/InvalidLinkBear.py

```diff
@@ -43,8 +43,9 @@ def get_status_code(url, timeout):
     def find_links_in_file(file, timeout, link_ignore_regex):
```

Comment:
This cannot be done in 50 characters. (Because the filename occupies some space too) What should I do ? 
``InvalidLinkBear.py: Edit regex ignore placeholders``
Is this fine ?